web/src/components/Plugin/PluginPage.tsx

```diff
@@ -92,6 +92,13 @@ const PluginPage: React.FC<Props> = ({
       form.setFieldsValue({ plugin_config_id });
     });
   }, []);
+  const openPlugin = pluginList.filter(
```

Comment:
`openPlugin` is Not a good variable name, It should be a list, for example: `openPluginList`